src/protocols/bdx/BdxTransferSession.h

```diff
@@ -146,39 +147,33 @@ class DLL_EXPORT TransferSession
 
     /**
      * @brief
-     *   Indicates the presence of pending output and includes any data for the caller to take action on.
-     *
-     *   This method should be called frequently in order to be notified about any messages received. It should also be called after
-     *   most other methods in order to notify the user of any message that needs to be sent, or errors that occurred internally.
-     *
-     *   It is possible that consecutive calls to this method may emit different outputs depending on the state of the
-     *   TransferSession object.
-     *
-     *   Note that if the type outputted is kMsgToSend, the caller is expected to send the message immediately, and the session
-     *   timeout timer will begin at curTime.
+     *   Callback that the session calls to notify the caller about any messages received or messages that need to be send
+     *   or errors that occurred internally.
      *
      *   See OutputEventType for all possible output event types.
      *
-     * @param event     Reference to an OutputEvent struct that will be filled out with any pending output data
-     * @param curTime   Current time
+     * @param event     Reference to an OutputEvent struct that will be filled out with
+     *                  the generated output event
      */
-    void PollOutput(OutputEvent & event, System::Clock::Timestamp curTime);
+    using OutputEventCallback = void (*)(void * context, OutputEvent & event);
```

Comment:
What is the "context" param here?  Should be documented.

Reply:
done.